src/webui/api/torrentscontroller.cpp

```diff
@@ -262,20 +284,20 @@ void TorrentsController::infoAction()
     for (const QString &hash : hashes)
         hashSet.insert(BitTorrent::InfoHash {hash});
 
+    const TorrentFilter torrentFilter(filter, (hashes.isEmpty() ? TorrentFilter::AnyHash : hashSet), category);
     QVariantList torrentList;
-    TorrentFilter torrentFilter(filter, (hashes.isEmpty() ? TorrentFilter::AnyHash : hashSet), category);
     for (BitTorrent::TorrentHandle *const torrent : asConst(BitTorrent::Session::instance()->torrents()))
     {
         if (torrentFilter.match(torrent))
             torrentList.append(serialize(*torrent));
     }
 
     std::sort(torrentList.begin(), torrentList.end()
```

Comment:
Ideally `torrentList` should become `QJsonArray` type. However, I couldn't get it work with `std::sort` and `torrentList.mid()` won't work too.